plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/loadInterpreter.kt

```diff
@@ -91,8 +100,16 @@ import org.jetbrains.kotlinx.dataframe.plugin.impl.api.ToDataFrameFrom
 import org.jetbrains.kotlinx.dataframe.plugin.impl.api.ToTop
 import org.jetbrains.kotlinx.dataframe.plugin.impl.api.Update0
 import org.jetbrains.kotlinx.dataframe.plugin.impl.api.UpdateWith0
+import org.jetbrains.kotlinx.dataframe.plugin.utils.Names
 
+@OptIn(UnresolvedExpressionTypeAccess::class)
 internal fun FirFunctionCall.loadInterpreter(session: FirSession): Interpreter<*>? {
+    if (
+        calleeReference.toResolvedNamedFunctionSymbol()?.callableId == Names.TO &&
```

Comment:
so.. does this mean it fails for `dataFrameOf(Pair("name", listOf(1, 2, 3)))`?

Reply:
It's probably fine, most people will use `to` anyway, but it might be something we should mention in a doc somewhere, for that one person who's going to be very confused as to why their Pair is not recognized by the plugin if the function only accepts pairs as the type argument.

Reply:
Good point. Let's support Pair after i put in place more generic way for handling stdlib functions other than "if". And indeed with such things it's much easier to run into something plugin won't understand. One of the things that can be addressed with custom Checker that will highlight it as warning